python/ray/rllib/ppo/ppo.py

```diff
@@ -181,19 +184,19 @@ def standardized(value):
             loss = np.mean(loss)
             policy_loss = np.mean(policy_loss)
             vf_loss = np.mean(vf_loss)
-            kl = np.mean(kl)
+            kl = np.mean(kl, axis=0)
             entropy = np.mean(entropy)
             sgd_end = time.time()
             print(
                 "{:>15}{:15.5e}{:15.5e}{:15.5e}{:15.5e}{:15.5e}".format(
-                    i, loss, policy_loss, vf_loss, kl, entropy))
+                    i, loss, policy_loss, vf_loss, np.sum(kl), entropy))
 
             values = []
             if i == config["num_sgd_iter"] - 1:
                 metric_prefix = "ppo/sgd/final_iter/"
                 values.append(tf.Summary.Value(
                     tag=metric_prefix + "kl_coeff",
-                    simple_value=self.kl_coeff))
+                    simple_value=self.kl_coeff[0]))
```

Comment:
mean?

Reply:
I'm actually unclear what we want to return here; the mean of the kl of a list of agents isn't informative.